samples/JsxTranspiler/JsxTranspiler.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net462</TargetFrameworks>
+    <TargetFrameworks>net6.0;net472</TargetFrameworks>
```

Comment:
Please also update the conditions later in this file (i.e. PolySharpIncludeGeneratedTypes).